scripts/config_template.yaml

```diff
@@ -1,15 +1,19 @@
 # The data sets to load and how to interpret them
 data:
   data_set_1:
-    group_cols:
-      geography: region
-    rollout: 2023-09-01
+    rollout: [2023-09-01, 2024-09-01]
```

Comment:
I'm imagining some weird edge case where we inadvertently put in a rollout that is after the first data point in a season and get weird negative stuff

Reply:
Could happen. But it is the user's responsibility to know that information. Maybe this calls for an `assert` inside `insert_rollout` that checks that the provided rollout date(s) are at least sensible.